src/App.tsx

```diff
@@ -1,42 +1,32 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tab } from './types';
+import { Tabs } from './components/Tabs';
+import { tabs } from './tabs';
 
-export const tabs = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-];
+export const App: FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const onTabSelected = (tab: Tab) => {
+    if (tab.id !== selectedTabId) {
+      setSelectedTabId(tab.id);
+    }
```

Comment:
I always do it this way. I like to avoid nesting as much as possible.
```suggestion
    if (tab.id === selectedTabId) return;
    
    setSelectedTabId(tab.id);
```

Reply:
that unfortunately I can't do, linter won't let me, but I can do this:
`  const onTabSelected = (tab: Tab) => {
    if (tab.id === selectedTabId) {
      return;
    }

    setSelectedTabId(tab.id);
  };`